docs/api/classifier/class_prediction_error.rst

```diff
@@ -77,12 +84,37 @@ By contrast, in the following example, the ``RandomForestClassifier`` does a gre
     # Draw visualization
     visualizer.show()
 
+Quick Method
+------------
+
+Similar functionality as above can be achieved in one line using the associated quick method, ``class_prediction_error``. This method will instantiate and fit a ``ClassPredictionError`` visualizer on the training data, then will score it on the optionally provided test data (or the training data if it is not provided).
+
+.. plot::
+    :context: close-figs
+    :alt: class_prediction_error quick method
+
+    from sklearn.svm import LinearSVC
+    from sklearn.model_selection import train_test_split as tts
+    from yellowbrick.classifier import class_prediction_error
+    from yellowbrick.datasets import load_occupancy
+
+    # Load the dataset and split into train/test splits
+    X, y = load_occupancy()
+    X_train, X_test, y_train, y_test = tts(
+        X, y, test_size=0.2, shuffle=True
+    )
+
+    class_prediction_error(
+        LinearSVC(random_state=42),
+        X_train, y_train, X_test, y_test,
+        classes=["vacant", "occupied"]
```

Comment:
Great example! I only added the `classes` argument here to show users that they could still use the regular params in the quick method.